scheduling-api-http/src/integration-test/java/org/ow2/proactive/scheduling/api/GraphqlServiceIntegrationTest.java

```diff
@@ -261,7 +262,35 @@ public void testQueryJobsFilterByContainsNames() {
     @Rollback
     @Test
     @Transactional
-    public void testQueryJobsFilterByOwners() {
+    public void testQueryJobsFilterByContainNames() {
+        addJobData(10);
+
+        Map<String, Object> queryResult = executeGraphqlQuery(String.format("{ jobs(%s:{%s:\"*ob*\"}) { edges { cursor node { id name } } } }",
+                                                                            FILTER.getName(),
+                                                                            NAME.getName()));
+
+        List<?> jobNodes = (List<?>) getField(queryResult, "data", "jobs", "edges");
+        assertThat(jobNodes).hasSize(10);
+    }
+
+    @Rollback
+    @Test
+    @Transactional
+    public void testQueryJobsFilterByNotContainNames() {
+        addJobData(10);
+
+        Map<String, Object> queryResult = executeGraphqlQuery(String.format("{ jobs(%s:{%s:\"!*ob*\"}) { edges { cursor node { id name } } } }",
+                                                                            FILTER.getName(),
+                                                                            NAME.getName()));
+
+        List<?> jobNodes = (List<?>) getField(queryResult, "data", "jobs", "edges");
+        assertThat(jobNodes).hasSize(0);
```

Comment:
it would be better that the test actually return some results (instead of 0)

Reply:
Ok